express/api_routes.js

```diff
@@ -30,8 +33,12 @@ function setupProxyRoutes() {
         }
     });
     
-    apiRoutes.use(sessionMiddleware(true), async (req, res, next) => {
+    apiRoutes.use(["/", "/:cluster_id"], sessionMiddleware(extension, true), async (req, res, next) => {
         try {
+            const clusterId = req.params.cluster_id;
+            if (clusterId) {
+                extension = ExtensionFactory.getExtension(clusterId)
+            }
             const client = await extension.getPlatformClient(req.fdkSession.company_id, req.fdkSession);
             req.platformClient = client;
             req.extension = extension;
```

Comment:
req.clusterId = clusterId;

Injection will be helpful

Reply:
this is already available on request params. Other routes will also have this integrated anyways since it is part of launch url itself. 

Eventually we will need to stop attaching things directly to request object as many frameworks restrict it. So rather we are skipping something already available natively.